src/pages/api/webhooks/saleor/PRODUCT_CREATED.ts

```diff
@@ -0,0 +1,19 @@
+import { SaleorAsyncWebhook } from "@saleor/app-sdk/handlers/next";
+import { ProductEditedDocument, ProductEditedSubscription } from "../../../../../generated/graphql";
+import { saleorApp } from "../../../../../saleor-app";
+import { handler } from "./_index";
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
+
+export const webhook = new SaleorAsyncWebhook<ProductEditedSubscription["event"]>({
```

Comment:
```suggestion
export const webhookProductCreated = new SaleorAsyncWebhook<ProductEditedSubscription["event"]>({
```

Maybe rename it to avoid aliases on imports in manifest?